CHANGELOG.md

```diff
@@ -1,14 +1,17 @@
 # Release 0.16.0
 ### Improvements
 
+* Adds the compatibility tag for Python 3.9.

```

Comment:
It looks that the tag was already there in `setup.py` from the previous release, so maybe it's okay just to remove the entry in here.

Reply:
Ok I remove it!